src/screens/benefit/Details.tsx

```
@@ -49,35 +48,45 @@ interface BenefitItem {
   email: string;
 }
 
 interface WebFormProps {
   url?: string;
-  formData?: AuthUser;
-}
-interface Context {
-  bpp_id?: string;
-  bap_uri?: string;
-  // Add other fields as necessary
+  formData?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)